### DISCUSSION TITLE
馬斯克預估：2050 年家家戶戶可能都有人形機器人

### DISCUSSION THREAD

**Discussion by 110409538** [1101-GS4719-C/Python-Community-GS4719-C#575] [科技新知]
> 如今像波士頓動力（Boston Dynamics）等公司已經向世人展現擁有驚人行動能力的人形機器人，但家用 AI 機器人的普及仍有一段路要走。目前人形機器人最缺乏的是具備足夠的智慧，亦即機器人即使在沒有明確指示的情況下，也能在真實世界裡四處走動並協助處理各種有用事情的能力。

>  馬斯克創立 Neuralink 公司的部分原因是為了「減輕 AI 的威脅」。為了實現 AI 機器人的安全性，馬斯克認為必須確保本地化 ROM 晶片無法 進行OTA（Over the Air）無線更新，如此一來，任何人都可以在必要的時候直接命令機器人停止動作。除此之外，設立針對 AI 的監管機構也很重要。
      
> 在成本上，馬斯克強調，儘管人形機器人的最初成本直到量產製造趕上來並壓低成本之前會處於居高不下的狀態，但人形機器人成本還是有可能比汽車低。接下來，我們一定會看到人形機器人的用途逐年快速成長、成本逐漸下降，量產規模也會隨之快速擴大。

> 蠻期待以後家裡也有機器人會是怎樣的感覺,這樣就不用做家務了,不知道能不能幫忙洗碗:thinking:

新聞來源:https://technews.tw/2022/04/20/robot-at-home/

**Comment by 11timyyy** [comment]
我認為讓機器人停止動作這個機制非常重要，一定要控制住機器人。

**Comment by PoooYen** [comment]
機器人究竟可以取代人類的工作到甚麼地步真是既期待又怕受傷害呢

**Comment by yi-1111** [comment]
真是令人期待的未來科技

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，期待未來科技的發展。

**Comment by cordydeng** [comment]
隨著科技的進步，機器人也逐漸普及，不過一般人應該不會想要一個人形機器人占空間吧?

**Comment by lx3211** [comment]
去年有一部「天兵阿榮」是在講用機器人來社交的電影，內容滿有趣而且值得反思。
